src/lib.js

```diff
@@ -189,6 +189,9 @@ function sendSlackMessage(webhookUrl,payload) {
           path: wh.pathname,
         },
         function(resp) {
+          // throw away response
+          resp.on('data',function() {});
```

Comment:
Ensure response data is handled - otherwise Node.js won't end/finish up.

We don't need the response body (HTTP status code is enough) - so simply junk the data received.

Reply:
Is this a known bug? Seems like a pretty basic requirement to fire-and-forget requests!

Reply:
I don't believe so @jack-green - as we're a few point releases into `v20` now and assume it's "desired". But without any code changes, moving between `v18` -> `v20` - the HTTPS request is made (and Slack message posted) - but the Node process will then "hang" indefinitely - so I can only assume it's a latent Promise awaiting the resolve? 

In GitHub Actions land - the action is forcefully killed after three minutes - due to this inactivity - which is nice - but isn't a solution :)